packages/iTwinUI-react/src/types/custom.d.ts

```diff
@@ -3,6 +3,7 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 declare module '*.svg' {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const content: any;
```

Comment:
Might be possible to use `unknown` here. But minor because this is removed in #660

Reply:
These can be removed now

Reply:
you mean whole file?

Reply:
removed